src/common/snippets/src/lowered/loop_manager.cpp

```diff
@@ -438,6 +463,31 @@ void LinearIR::LoopManager::fuse_loops(LinearIR::constExprIt loop_begin_target,
     remove_loop_info(from);
 }
 
+std::vector<lowered::pass::PassPipeline> LinearIR::LoopManager::fuse_loop_handlers(
+    std::vector<lowered::pass::PassPipeline>& from,
+    std::vector<lowered::pass::PassPipeline>& to) {
+    const auto min_size = std::min(from.size(), to.size());
+    std::vector<lowered::pass::PassPipeline> merged_handlers;
+    merged_handlers.resize(min_size);
+    for (size_t i = 0; i < min_size; ++i) {
```

Comment:
By the way, should we control the order of passes?
Seems like we push back all different passes from second pipeline.

Reply:
Sorry, didn't get your point, could you please clarify a bit? Or we can discuss the question offline

Reply:
Discussed offline: the current implementation doesn't support the merging of 2 pass pipelines which contain passes whose running order is important. However, such support isn't needed at the moment: all passes, which are used as loop specific iteration handler, have no running order restrictions.
Nevertheless, I agree that the current limitation of the `merge_pipelines` method should be documented - I'll update the method description